### PR TITLE
EnumerationGraphType supports GraphQLAttributes

### PR DESCRIPTION
**Pull request by Shane32**
@sungam3r You may wish to review or incorporate this during your enum rework

### CODE REVIEW

File: src/GraphQL/Types/Scalars/EnumerationGraphType.cs [hunk 1]
```diff
@@ -140,16 +140,41 @@ public EnumerationGraphType()
...
+                {
+                    Name = name,
+                    Value = value,
+                    Description = description,
+                    DeprecationReason = deprecation,
+                };
+                bool ignore = false;
+                foreach (var attr in member.GetCustomAttributes<GraphQLAttribute>())
+                {
+                    if (!attr.ShouldInclude(member, true))
```
Comment (sungam3r) [inline src/GraphQL/Types/Scalars/EnumerationGraphType.cs:163]: true/false? null? for second argument

**Review by sungam3r — APPROVED** [review]

File: src/GraphQL/Types/Scalars/EnumerationGraphType.cs (hunk 1 above)
Comment (Shane32) [reply src/GraphQL/Types/Scalars/EnumerationGraphType.cs:163]: As a scalar is both input and output equally. Do you think I should make the property a nullable bool or just write in the xml comment that it’s true for enums?

Comment (Shane32) [reply src/GraphQL/Types/Scalars/EnumerationGraphType.cs]: Changing to null.

Comment (sungam3r) [reply src/GraphQL/Types/Scalars/EnumerationGraphType.cs]: ok

**Pull request merged**